src/main/java/seedu/address/logic/commands/ExitCommand.java

```diff
@@ -8,7 +8,8 @@
  */
 public class ExitCommand extends Command {
 
-    public static final String COMMAND_WORD = "exit";
+    public static final String COMMAND_WORD_EXIT = "exit";
+    public static final String COMMAND_WORD_QUIT = "quit";
```

Comment:
You should not name your constants to match their content, as this makes them quite redundant and is not resistant to change. Instead, do something like `COMMAND_WORD` and `COMMAND_WORD_ALIAS`.